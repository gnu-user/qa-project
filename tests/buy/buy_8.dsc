TITLE
    Buy Amount Prompt
DESCRIPTION
    Run the login transaction using an admin account. Initiate the buy 
    transaction. When prompted, enter a title which exists. Verify that the 
    user is prompted to enter the number of tickets for purchase.
DEPENDENCIES
    buy_7
