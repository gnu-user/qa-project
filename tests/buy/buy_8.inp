login
administrator
buy
The Godfather