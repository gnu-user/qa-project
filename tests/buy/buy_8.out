Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the event title:
Enter the number of tickets to purchase: 